src/main/java/gregtech/common/covers/detector/CoverDetectorFluidAdvanced.java

```diff
@@ -162,34 +174,34 @@ public void update() {
     @Override
     public void writeToNBT(@NotNull NBTTagCompound tagCompound) {
         super.writeToNBT(tagCompound);
-        tagCompound.setInteger("min", this.min);
-        tagCompound.setInteger("max", this.max);
+        tagCompound.setLong("min", this.min);
+        tagCompound.setLong("max", this.max);
         tagCompound.setBoolean("isLatched", this.isLatched);
         tagCompound.setTag("filter", this.fluidFilter.serializeNBT());
     }
 
     @Override
     public void readFromNBT(@NotNull NBTTagCompound tagCompound) {
         super.readFromNBT(tagCompound);
-        this.min = tagCompound.getInteger("min");
-        this.max = tagCompound.getInteger("max");
+        this.min = tagCompound.getLong("min");
+        this.max = tagCompound.getLong("max");
```

Comment:
Do you know if this will mess up existing setups? What I mean is that previously min/max were ints, and I don't know if `readLong` will see the Int values that were previously there, or if it will only look for a new tag. I think it should be ok, because of using the same key, but I am not sure.